builder/proxmox/common/config.go

```diff
@@ -84,6 +84,7 @@ type additionalISOsConfig struct {
 	Device                string `mapstructure:"device"`
 	ISOFile               string `mapstructure:"iso_file"`
 	ISOStoragePool        string `mapstructure:"iso_storage_pool"`
+	ISODownloadPVE        bool   `mapstructure:"iso_download_pve"`
```

Comment:
This would benefit from documentation to make it clear what the behaviour should be if defined.

Reply:
added it to "clone" documentation as it already is in the "iso" part

Reply:
this variable name is somewhat ambiguous, can you please rename it to `iso_download_from_pve_node` (the `node` word is used to match the proxmox-iso source `node` property)?